janitor/timeseries.py

```diff
@@ -0,0 +1,123 @@
+"""
+Time series-specific data cleaning functions.
+"""
+
+import pandas as pd
+import pandas_flavor as pf
+from pandas.api.types import is_datetime64_any_dtype as is_datetime
+from collections import namedtuple
+from janitor import check
+
+
+def _flag_missing_timestamps(
+    df: pd.DataFrame,
+    frequency: str,
+    column_name: str,
+    first_time_stamp: pd.Timestamp,
+    last_time_stamp: pd.Timestamp,
+) -> namedtuple:
+    """
+    Utility function to test if input data frame is missing any timestamps relative to expected timestamps
+    generated based on the first_time_stamp, last_time_stamp and frequency.
```

Comment:
First line should be a concise description. 

Here's a proposed change:

```suggestion
    Flag timestamps that  are missing.

    Utility function to test if input data frame is missing any timestamps
    relative to expected timestamps 
    generated based on the first_time_stamp, last_time_stamp and frequency.
```

Also incorporated [semantic line breaks](https://sembr.org/) into that line. I probably didn't include this in the contribution docs, though I should. Would you be kind enough to fix the others? Using best judgment here is good enough, don't need to be _too_ strict about it.